lib/dynflow/persistence_adapters/sequel.rb

```diff
@@ -124,7 +124,7 @@ def find_old_execution_plans(age)
       def find_past_delayed_plans(time)
         table_name = :delayed
         table(table_name)
-          .where(::Sequel.lit('start_at <= ? OR (start_before IS NOT NULL AND start_before <= ?)', time, time))
+          .where(::Sequel.lit('start_at <= ? OR (start_before IS NOT NULL AND start_before <= ?) and frozen != \'t\'', time, time))
```

Comment:
I wasn't sure exactly if this was needed, but it seemed like it.

Reply:
Do all the DBs we support use the same value (`'t'`) to store truthy values? Maybe turning this into something like the following would be safer

```ruby
table(table_name)
  .where(:frozen => false)
  .where(::Sequel.lit('start_at <= ? OR (start_before IS NOT NULL AND start_before <= ?)', time, time))
```

Reply:
I assume you meant something like:
```
          .where(::Sequel.lit('start_at <= ? OR (start_before IS NOT NULL AND start_before <= ?) and frozen != ?', time, time, true))
```

Reply:
added

Reply:
Chained where conditions should get joined by ands into one, but I'm ok with your solution

Reply:
i see, i misread your comment.  Thanks!